dm/pkg/ha/bound_test.go

```diff
@@ -75,20 +75,23 @@ func (t *testForEtcd) TestSourceBoundEtcd(c *C) {
 	c.Assert(len(errCh), Equals, 0)
 
 	// get bound1 back.
-	sbm2, rev4, err := GetSourceBound(etcdTestCli, worker1)
+	sbm2, rev4, err := GetSourceBound(etcdTestCli, worker1, "")
```

Comment:
As `PutSourceBound` supports new key, maybe can add some ut for `GetSourceBound`'s source parameter.

Reply:
addressed in https://github.com/pingcap/tiflow/pull/4685/commits/0eccb60e0d750c22a48c1ee7d2dd6018bf493cd1
besides this part should cover some of the tests